src/math/Interpolant.js

```diff
@@ -47,7 +47,7 @@ Interpolant.prototype = {
 
 		validate_interval: {
 
-			seek: {
+			seek: while(true) {
```

Comment:
I don't think so. I'm coming from a java background and you can only really break from loops. I can do make an edit and see what it "says" I'll go do that really quick on my local copy and let you know.